arch/arm64/core/isr_wrapper.S

```diff
@@ -46,6 +46,17 @@ SECTION_FUNC(TEXT, _isr_wrapper)
 	bl	z_soc_irq_get_active
 #endif /* !CONFIG_ARM_CUSTOM_INTERRUPT_CONTROLLER */
 
+	/*
+	 * Ignore Special INTIDs 1020..1023 see 2.2.1 of Arm Generic Interrupt Controller
+	 * Architecture Specification GIC architecture version 3 and version 4
+	 */
+	cmp	x0, 1019
+	b.le	oob
+	cmp	x0, 1023
+	b.gt	oob
+	b	spurious_continue
```

Comment:
This is something GIC specific so either we guard this with some `ifdef` or (better) we move this to same GIC specific place (i.e. drivers/interrupt_controller/intc_gicv3.c)

Reply:
Linux does that in the driver, but we should still have a way to detect an irq to ignore in asm